rfc/rfc002-authentication-token.md

```diff
@@ -401,3 +401,129 @@ Beside the mandatory VP fields, the following applies:
+The care organisation uses the information of the current user session to present the user with a statement about its identity and asks the user to confirm this.
+After confirmation this bundle of information is singed with the care organisation's private key.
+
+This method makes it possible to assert a user identity without the need for a personal authentication means.
+This puts the care organisation in the role of identity issuer instead of a trusted third party.
+A use-case (Bolt) can choose to allow this authentication under certain (low risk) circumstances which still require a person to be authenticated in order to comply with the applicable legislation e.g. the [NEN7513](https://www.nen.nl/nen-7513-2018-nl-245399) (Dutch standard for logging in context of medical ICT).
+
+Note: The _means_ is designed in a way that it is interchangeable with (other) means with a higher trust level, (using a personal authentication device).
+Even it is technically possible, this means MUST NOT be used to perform machine to machine interactions which are not initiated by the current user, e.g. background jobs, fetching data automatically etc.
+By doing so, a use-case (Bolt) can not update the trust level of a means without braking functionality.
+This means MUST only be used to claim the identity of actual and identifying natural persons working under the responsibility of the care organisation, and MUST NOT identify e.g. shared accounts, teams, administrators, dummy or test users.
+
+#### 7.3.1 Attributes
+
+In order for the implementation to sign the contract the following information is needed:
+
+1. The `challenge` in the form of a login contract.
+2. The `credential` which contains the following information:
+   * The `@context` which MUST contain the following urls:
+      1. `http://schema.org/`: contains the Organisation, employeeRole and person types.
+      2. `https://nuts.nl/credentials/v1`: contain definition of the `NutsEmployeeIdentityCredential` type.
+      3. `https://www.w3.org/2018/credentials/v1`: contains definition of verifiable credential fields.
+   * The `type` which MUST contain the `VerifiableCredential` and `NutsEmployeeIdentityCredential` types.
+   * The `issuer` which contains the DID of the care organisation. This DID MUST resolve to a DID Document which contains the public key used to sign the credential.
+   * The `credentialSubject` which contains the following information:
+      1. The `@type` which contains the `EmployeeRole` type to indicate the contents describes a EmployeeRole as defined by schema.org
+      2. The `identifier` (optional) which contains the employee number or username. This identifier MUST uniquely identify one natural person within the care organisation. If this number is not available, the email address field of the `Person` MUST be used.
+      3. The `roleName` (optional) which contains the name of the role of the person during this user session.
```

Comment:
Why add this? Is the verifying party supposed to do something with it? If so, is it standardized? (if not, it's probably useless for the verifying party).